docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -91,11 +92,31 @@ recommended when executing large batch queries, as the cluster can more
 efficiently retry smaller tasks within the query rather than retry the whole
 query.
 
-``TASK`` retry policy requires a configured :ref:`exchange manager
-<fte-exchange-manager>` to store spooled exchange data used for each task. It is
-also strongly recommended to set the ``query.low-memory-killer.policy``
-configuration property to ``total-reservation-on-blocked-nodes``, or queries may
-need to be manually killed if the cluster runs out of memory.
+The following cluster configuration changes are recommended to improve the
+efficacy of fault-tolerant execution with a ``TASK`` retry policy:
+
+* Set the ``query.low-memory-killer.policy``
+  :doc:`query management property </admin/properties-query-management>` to
+  ``total-reservation-on-blocked-nodes``, or queries may
```

Comment:
This is the default, right?

Reply:
If the docs are correct, then yes. Still worth calling out I think, because it's a property that a reader might not associate with FTE and have configured differently.